adoc/bp_chap_expanded_support.adoc

```diff
@@ -383,3 +383,130 @@ See the matrix of {productname} clients on the main page of the {productname} wi
 {prompt.root}systemctl restart salt-minion
 ----
 . Proceed to menu:Main Menu[Salt > Keys] from the {webui} and accept the minion's key.
+
+
+
+[[bp.expanded-support.ubuntu]]
+== Managing {ubuntu} Clients
+
+
+Support for {ubuntu} Clients was added in {susemgr} 3.2.
+Currently, Salt minions running {ubuntu} 16.04 LTS and 18.04 are supported.
+
+Bootstrapping is supported for starting {ubuntu} clients and performing initial state runs such as setting repositories and performing profile  updates.
```

Comment:
Nitpick: double space between profile and updates

Reply:
Wow, good spot!